app/Filament/Resources/FlightResource.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace App\Filament\Resources;
+
+use App\Filament\Resources\FlightResource\Pages;
+use App\Models\Flight;
+use Filament\Forms;
+use Filament\Forms\Form;
+use Filament\Resources\Resource;
+use Filament\Tables;
+use Filament\Tables\Columns\TextColumn;
+use Filament\Tables\Table;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\SoftDeletingScope;
+
+class FlightResource extends Resource
+{
+    protected static ?string $model = Flight::class;
+
+    protected static ?string $navigationGroup = 'operations';
+    protected static ?int $navigationSort = 2;
+
+    protected static ?string $navigationLabel = 'flights';
+
+    protected static ?string $navigationIcon = 'heroicon-o-adjustments-vertical';
+
+    protected static ?string $recordTitleAttribute = 'name';
+
+    public static function form(Form $form): Form
+    {
+        return $form
+            ->schema([
+                //
+            ]);
+    }
+
+    public static function table(Table $table): Table
+    {
+        return $table
+            ->columns([
+                TextColumn::make('ident')->label('Flight #'),
+                TextColumn::make('dpt_airport_id')->label('Dep')->searchable(),
+                TextColumn::make('arr_airport_id')->label('Arr')->searchable(),
+                TextColumn::make('dpt_time')->label('Dpt Time'),
+                TextColumn::make('arr_time')->label('Arr Time'),
+                TextColumn::make('notes')->label('Notes'),
+                TextColumn::make('active')->label('Active')->badge()->color(fn ($record) => $record->active ? 'success' : 'danger')->formatStateUsing(fn (bool $state): string => $state ? 'Active' : 'Inactive'),
```

Comment:
Maybe we can use heroicons here too ? Since we only have active/inactive

Reply:
I tried it, but for a unknown (yet) reason, when i try to show only the heroicon it don't show anything for this column

Reply:
I tried using IconColumn instead of TextColumn, works well. I also used circle icons instead of "normal". Here is the code `IconColumn::make('visible')->label('Visible')->color(fn ($record) => $record->visible ? 'success' : 'danger')->icon(fn ($record) => $record->visible ? 'heroicon-o-check-circle' : 'heroicon-o-x-circle')`. This works for both active and visible columns.